ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/views/PaywallView.kt

```diff
@@ -89,6 +95,7 @@ class PaywallView : FrameLayout {
      */
     fun setOfferingId(offeringId: String?) {
         this.offeringId = offeringId
+        this.updateOfferingId()
         invalidate()
```

Comment:
I don't think we need this anymore, I tested it without the invalidate and it works